deckhand/conf/config.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2017 AT&T Intellectual Property.  All other rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from oslo_config import cfg
+
+barbican_group = cfg.OptGroup(
```

Comment:
Since this is a "skeleton" placeholder, it may make sense to start thinking about what is the external infrastructure this will need to communicate with.  Clearly, a database.  Message queue infrastructure? Memcache? Even if they are also empty groups it starts shaping this up.